3rdParty/s2geometry/CMakeLists.txt

```diff
@@ -3,6 +3,10 @@
 set (PYTHON_INCLUDE_DIR "")
 project(s2-geometry C CXX)
 
+if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
```

Comment:
will this also work for #include statements?

Reply:
Sadly, no. This can only suppress warnings in the s2 cpp files, not when we include the headers somewhere.